src/js/actions.js

```diff
@@ -76,7 +75,7 @@ export function setViewedPages(pageKeys) {
   };
 }
 
-function submitToUrl(body, submitUrl, recordEvent) {
+export function submitToUrl(body, submitUrl, recordEvent) {
```

Comment:
This is pretty generic now but potentially could be used by someone with plain-vanilla needs.